trackerquery_blackbox_test.go

```diff
@@ -0,0 +1,201 @@
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           "",
+		},
+		// Update tracker query API with different parameters
+		{
+			method:             "PUT",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           getExpiredAuthHeader(t, privatekey),
+		}, {
+			method:             "PUT",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           getMalformedAuthHeader(t, privatekey),
+		}, {
+			method:             "PUT",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           getValidAuthHeader(t, differentPrivatekey),
+		}, {
+			method:             "PUT",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           "",
+		},
+		// Delete tracker query API with different parameters
+		{
+			method:             "DELETE",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           getExpiredAuthHeader(t, privatekey),
+		}, {
+			method:             "DELETE",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           getMalformedAuthHeader(t, privatekey),
+		}, {
+			method:             "DELETE",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           getValidAuthHeader(t, differentPrivatekey),
+		}, {
+			method:             "DELETE",
+			url:                "/api/trackerqueries/12345",
+			expectedStatusCode: 401,
+			expectedErrorCode:  "jwt_security_error",
+			payload:            createTrackerQueryPayload,
+			jwtToken:           "",
+		},
+		// Try fetching a random tracker query
+		// We do not have security on GET hence this should return 404 not found
+		{
+			method:             "GET",
+			url:                "/api/trackerqueries/088481764871",
+			expectedStatusCode: 404,
+			expectedErrorCode:  "not_found",
+			payload:            nil,
+			jwtToken:           "",
+		},
+	}
+}
+
+// This test case will check authorized access to Create/Update/Delete APIs
+func TestUnauthorizeTrackerQueryCUD(t *testing.T) {
```

Comment:
Can we please have one positive test case(happy test), Just add one more struct for create,update,delete in the test data like-
```javascript
          {
			method:             "POST",
			url:                "/api/trackerqueries",
			expectedStatusCode: 201,
			expectedErrorCode:  nil,
			payload:            createTrackerQueryPayload,
			jwtToken:           SOME_VALID_TOKEN,
		},
```
And then we can rename `TestUnauthorizeTrackerQueryCUD` to `TestAuthorizationForTrackerQueryCUD`.

Or such test case already exist elsewhere hence not included here ?